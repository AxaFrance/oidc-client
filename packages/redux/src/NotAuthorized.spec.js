// Link.react.test.js
import React from 'react';
import renderer from 'react-test-renderer';
import NotAuthorized from './NotAuthorized';

test('Render <NotAuthorized/> correctly', () => {
  const component = renderer.create(<NotAuthorized />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
