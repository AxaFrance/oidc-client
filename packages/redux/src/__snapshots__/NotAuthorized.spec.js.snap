// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render <NotAuthorized/> correctly 1`] = `
<div>
  <div
    className="container"
  >
    <h1>
      Autorisation
    </h1>
    <p>
      Vous n'êtes pas autorisé à accéder à cette ressource.
    </p>
  </div>
</div>
`;
